src/frontend/sharedComponents/Editor/index.tsx

```diff
@@ -124,7 +124,6 @@ export const Editor = ({
                           // TODO: Does it make sense to provide the `observationDocId` in this case?
                           // Reasoning for not doing so is because the photo isn't actually saved yet, so it's technically not
                           // officially associated with the observation being created/edited.
```

Comment:
would like some thoughts on this. for draft photos that are created while editing an observation, we do not show the associated observation ID for the reasoning mentioned in this comment. wondering if that should be reconsidered though.

Reply:
I think for simplicity lets leave it as is and if we get feedback from the field we can change it!